app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스")
+class TaskServiceTest {
+
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+    private static final String UPDATE_POSTFIX = "New";
```

Comment:
```suggestion
    private static final String TASK_TITLE = "test";
    private static final String UPDATE_POSTFIX = "New";
    
    private TaskService taskService;
```

상수랑 멤버 변수랑 구분해주면 좋을 것 같아요

Reply:
네 분리했습니다!